packages/core/EventManager.ts

```diff
@@ -168,7 +168,7 @@ export default class EventManager {
         meetingPassword: createdEventObj ? createdEventObj.password : result.createdEvent?.password,
         meetingUrl: createdEventObj ? createdEventObj.onlineMeetingUrl : result.createdEvent?.url,
         externalCalendarId: isCalendarType ? result.externalId : undefined,
-        credentialId: isCalendarType ? result.credentialId : undefined,
+        credentialId: result.credentialId ?? undefined,
```

Comment:
@alannnc Some video integrations (like this one) rely of `credentialId` in booking ref. I think this should be safe to do but just double-checking.

Reply:
This one looks okay to me.